.zenodo.json

```diff
@@ -263,7 +263,7 @@
     },
     {
       "affiliation": "CIBIT, UC",
-      "name": "Machado, F\u00e1tima",
+      "name": "Dias, Maria de Fatima",
```

Comment:
Could you also add an entry to [`.mailmap`](https://github.com/nipy/nipype/blob/master/.mailmap)? The format would be:

```
Maria de Fatima Dias <your.preferred@email.tld>
```

If you commit with multiple email addresses, you can add additional lines to ensure that all your commits are associated to a single `Name <email>`:

```
Maria de Fatima Dias <your.preferred@email.tld> <other.1@email.tld>
Maria de Fatima Dias <your.preferred@email.tld> <other.2@email.tld>
```